module/documents/item.mjs

```diff
@@ -1980,7 +1981,8 @@ export default class Item5e extends Item {
 
   /** @inheritdoc */
   async _preCreate(data, options, user) {
-    await super._preCreate(data, options, user);
+    let allowed = await super._preCreate(data, options, user);
+    if ( allowed === false ) return allowed;
```

Comment:
We're not re-assigning this anywhere, nor returning it. That might change later but for now we should inline it.

```suggestion
    if ( (await super._preCreate(data, options, user)) === false ) return false;
```